runtime/README.md

```diff
@@ -159,7 +156,7 @@ Table: `ECDSA384_SIGNATURE_VERIFY` output arguments
 | **Name** | **Type** | **Description**
 | -------- | -------- | ---------------
 | chksum   | u32      | Checksum over other output arguments, computed by Caliptra. Little endian.
-| result   | u32      | Result code. Little endian.
+| result   | u32      | If 0, ECDSA verify failed. Else verify succeeded.
```

Comment:
Doesn't 0 usually indicate success?

Reply:
I can change the "result" naming. This is supposed to just be a boolean (but isn't because of word alignment)

Reply:
Why not treat it like `DISABLE_ATTESTATION` or `CALIPTRA_FW_LOAD` and return no output arguments?

Reply:
I wanted to differentiate between "Something bad happened in firmware" and "The ECDSA signature was bad". But I suppose that can just be another status code.

Reply:
Yeah, separate status codes seem reasonable - would need the status register to report those anyways, right? Since this `result` field just conveys a bool.

Reply:
Done.